pkg/server/in_cluster_application.go

```diff
@@ -0,0 +1,42 @@
+package server
+
+import (
+	"github.com/codeready-toolchain/registration-service/pkg/application"
+	"github.com/codeready-toolchain/registration-service/pkg/application/service"
+	servicecontext "github.com/codeready-toolchain/registration-service/pkg/application/service/context"
+	"github.com/codeready-toolchain/registration-service/pkg/application/service/factory"
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/codeready-toolchain/registration-service/pkg/kubeclient"
+	"k8s.io/client-go/rest"
+)
+
+func NewInClusterApplication(config configuration.Configuration, options ...factory.Option) (application.Application, error) {
```

Comment:
Could you add some comments about this particular application implementation.

Reply:
Done.